gluten-core/src/main/scala/io/glutenproject/expression/DateTimeExpressionsTransformer.scala

```diff
@@ -76,8 +76,8 @@ case class DateDiffTransformer(
       // In CH backend, datediff params are ('day', startDate, endDate).
       Lists.newArrayList(ExpressionBuilder.makeStringLiteral("day"), startDateNode, endDateNode)
     } else {
-      // In the others, datediff params are (startDate, endDate).
-      Lists.newArrayList(startDateNode, endDateNode)
+      // In the others, datediff params are (endDate, startDate).
+      Lists.newArrayList(endDateNode, startDateNode)
```

Comment:
Is this because new version of Velox changed the arg order of datediff?

Reply:
Yes, but the new one has aligned parameter order as Spark.
> case class DateDiff(endDate: Expression, startDate: Expression)